tests/test_python_cli.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+set -ex
+
+# This only checks that the following commands do not throw errors.
+# It is expected that other tests cover correctness of the generated
+# proposals, this just checks the basic usability of the CLI.
+
+keygenerator.sh --help
+keygenerator.sh --name alice
+keygenerator.sh --name bob --gen-enc-key
+
+
+python -m ccf.proposal_generator --help
```

Comment:
Perhaps worth annotating some of these and include [from the docs](https://microsoft.github.io/CCF/master/members/proposals.html) directly?

Reply:
I was going to do this, but unlike our other annotations we actually want the output of these commands in the docs, not just the input command (or code snippet) itself. If we find a solution for running commands and capturing their output to insert during the sphinx build, then we should definitely replace the proposal_generator docs with this, but I'm not going to tackle that just now.